src/content/docs/mobile-monitoring/new-relic-mobile-flutter/monitor-your-flutter-application.mdx

```diff
@@ -0,0 +1,493 @@
+
+    </Callout>
+
+## Query log data 
+
+New Relic stores your Flutter logs as custom events. You can query these logs and build dashboards for them using this NRQL query:
+```
+ SELECT * FROM Mobile Dart Console Events SINCE 30 MINUTES AGO
+```
+
+For more info on NRQL queries, see [Introduction to NRQL](docs/query-your-data/nrql-new-relic-query-language/get-started/introduction-nrql-new-relics-query-language/#where).
+
+## What's next? 
+
+Check out these optional API calls to further customize how New Relic collects and reports data:
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Track errors and resources"
+  >
+Set up error tracking and resource tracking. 
+
+Because NewRelic Mobile Start calls `WidgetsFlutterBinding.ensureInitialized`, if you are not using NewRelic Mobile Start, you need to call this method prior to calling `NewrelicMobile.instance.startAgent`.
+
+```dart
+if (Platform.isAndroid) {
+    appToken = AppConfig.androidToken;
+} else if (Platform.isIOS) {
+    appToken = AppConfig.iOSToken;
+}
+
+Config config = Config(
+    accessToken: appToken,
+    analyticsEventEnabled: true,
+    networkErrorRequestEnabled: true,
+    networkRequestEnabled: true,
+    crashReportingEnabled: true,
+    interactionTracingEnabled: true,
+    httpResponseBodyCaptureEnabled: true,
+    loggingEnabled: true,
+    webViewInstrumentation: true,
+    printStatementAsEventsEnabled: true,
+    httpInstrumentationEnabled:true);
+
+// NewrelicMobile.instance.start(config, () {
+//   runApp(MyApp());
+// });
+
+runZonedGuarded(() async {
+    WidgetsFlutterBinding.ensureInitialized();
+    FlutterError.onError = NewrelicMobile.onError;
+    await NewrelicMobile.instance.startAgent(config);
+    runApp(MyApp());
+}, (Object error, StackTrace stackTrace) {
+    NewrelicMobile.instance.recordError(error, stackTrace);
+});
+```
+
+AppToken is platform specific. You need to generate separate tokens for Android and iOS apps.
+
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Track screen navigation events"
+  >
+
+1. In order to track navigation events, you'll need to add the `NewRelicNavigationObserver` to your MaterialApp, WidgetsApp, or CupertinoApp.
+
+```dart
+import 'package:newrelic_mobile/newrelic_navigation_observer.dart';
+
+//....
+
+2. Add a name to route settings: `RouteSettings(name: 'Your Route Name')`. The root route name / will be replaced by root "/" for clarity's sake:
+
+```dart
+ routes: { 
+    // When navigating to the "/" route, build the HomeScreen widget. 
+    '/': (context) => HomeScreen(), 
+    // When navigating to the "/second" route, build the SecondScreen widget. 
+    '/second': (context) => SecondScreen(), 
+  },
+
+MaterialApp(
+    navigatorObservers: [
+        NewRelicNavigationObserver(),
+    ],
+    // other parameters
+)
+```
```

Comment:
@ndesai-newrelic Looking at the sample you provided in Slack, it still doesn't have RouteSettings in it. Where does it fit in then? Please propose a suggestion here.